db/migrate/20150506195759_remove_avatar_background_from_project.rb

```diff
@@ -0,0 +1,6 @@
+class RemoveAvatarBackgroundFromProject < ActiveRecord::Migration
+  def change
+    remove_column :projects, :avatar, :text
+    remove_column :projects, :background_image, :text
```

Comment:
Only the test stargazing projects have them. I figured it'd just be easier to reupload them.